pkg/state/filter_test.go

```diff
@@ -341,4 +341,32 @@ routes:
 		})
 	})
 
+	Context("With DNS Resolver populated", func() {
+		BeforeEach(func() {
+			state = nmstate.NewState(`interfaces:
+  - name: eth1
```

Comment:
could we use the `firstSecondaryNic` variable here instead of using `eth1` directly? Because the e2e tests in OCP use another NIC name for the first secondary (from `FIRST_SECONDARY_NIC` env var)

Reply:
Hey, this is just a unit test and the names don't really matter

Reply:
Ah sorry. My fault